app/Models/Source.php

```
@@ -57,6 +59,17 @@ protected static function boot()
             }
         });
 
+        // Featuring a channel demotes the others. Without this the flag silently
+        // becomes "one of the featured ones", and which one wins depends on insertion
+        // order, which is exactly the ambiguity the flag replaced.
+        static::saved(function ($source) {
+            if ($source->is_featured && $source->wasChanged('is_featured')) {
+                static::where('id', '!=', $source->id)
```

PHPStan (2.2.6)

[error] Call to an undefined static method App\Models\Source::where().


(staticMethod.notFound)